### PR TITLE
Niston

### PR DESCRIPTION
**Pull request by niston**
Bunch of stuff

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,15 @@ build/
 .github/
 .vs/
 /ck/package/
-*.pex
\ No newline at end of file
+*.pex
+/f4se-plugin/CMakeFiles.old/3.25.0-rc1/CompilerIdCXX/Debug/CompilerIdCXX.tlog
```
Comment (dlinny-lag) [inline .gitignore:6]: can be collapsed by 
`CMakeFiles*/*`

File: f4se-plugin/src/Menu/NAFMenu/MenuHandlers/InventoryMenuHandler.h [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+			if (a)
+			{
+				bool hadSWIKeyword = a->HasKeyword(Data::Forms::ShowWornItemsKW);
+				if (!hadSWIKeyword)
+				{
+					a->ModifyKeyword(Data::Forms::ShowWornItemsKW, true);
+				}
+
+				Menu::IStateManager::activeInstance->CloseMenu();
+				Sleep(100);
```
Comment (dlinny-lag) [inline f4se-plugin/src/Menu/NAFMenu/MenuHandlers/InventoryMenuHandler.h]: Is it necessary to suspend thread execution?

Comment (niston) [reply f4se-plugin/src/Menu/NAFMenu/MenuHandlers/InventoryMenuHandler.h]: yes, but it probably can be done in a more elegant way.

Comment (niston) [reply f4se-plugin/src/Menu/NAFMenu/MenuHandlers/InventoryMenuHandler.h]: actually, after the commit to unpause the containermenu, the delay is no longer necessary.
I have removed it.

**Comment by Deweh** [comment]
I'm going to temporarily merge this into the contributions branch to make fixes & adjustments. Then once all is verified, I'll merge it to master.

**Pull request merged**